packages/openid4vc/src/openid4vc-issuer/repository/OpenId4VcIssuanceSessionRecord.ts

```diff
@@ -97,6 +98,18 @@ export type DefaultOpenId4VcIssuanceSessionRecordTags = {
   presentationAuthSession?: string
 }
 
+export interface OpenId4VcIssuanceSessionRecordTransactionData {
+  // The expected number of credentials that will be issued in this transaction
+  numberOfCredentials: number
+
+  // The format of the credential request that was used to create the issuance session
+  requestFormat?: CredentialRequestFormatSpecific
```

Comment:
same here, do we need to store this?

Reply:
See https://github.com/openwallet-foundation/credo-ts/pull/2350#discussion_r2266804133